Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Model.rst

```diff
@@ -39,8 +39,16 @@ Or public getters:
 .. include:: /CodeSnippets/Extbase/Domain/ModelWithPublicGetters.rst.txt
 
 A public getter takes precedence over a public property. Getters have the
-advantage that you can make the properties themselves private and decide
-which ones should be mutable. It is also possible to have getters for
+advantage that you can make the properties themselves protected and decide
+which ones should be mutable.
+
+.. note::
+   Making properties :php:`private` does not work in Extbase models: The super
+   classes need to access the models properties directly. If your model must
```

Comment:
I assume, you mean the properties of one model? So either "model's properties" or "properties of the model"